addok/ds.py

```diff
@@ -47,7 +47,12 @@ def on_load():
     if config.DOCUMENT_STORE == RedisStore:
         params = config.REDIS.copy()
         params.update(config.REDIS.get('documents', {}))
-        _DB.connect(host=params['host'], port=params['port'], db=params['db'])
+        _DB.connect(
+            host=params['host'],
+            port=params['port'],
+            db=params['db'],
+            password=params.get('password', None)
```

Comment:
`dict.get()` will return `None` as default fallback, I think it's redundant to pass it explicitly here.